src/bulk_insert/bulk_insert.c

```diff
@@ -108,6 +106,7 @@ int _BulkInsert_ProcessNodeFile(RedisModuleCtx *ctx, GraphContext *gc, const cha
 		Graph_CreateNode(gc->g, label_id, &n);
 		for(unsigned int i = 0; i < prop_count; i++) {
 			SIValue value = _BulkInsert_ReadProperty(data, &data_idx);
+			if (SI_TYPE(value) == T_NULL) continue; // Don't insert null values.
```

Comment:
Please elaborate on this comment, as this is not RedisGraph optimization but Cypher restriction.